lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -385,6 +385,130 @@ def export_json(content_view_version, repositories)
       end
     end
 
+    class ImportCommand < HammerCLIForeman::Command
+      include HammerCLIForemanTasks::Async
+      include HammerCLI::Messages
+      include HammerCLIKatello::LocalHelper
+      include HammerCLIKatello::ApipieHelper
+
+      attr_accessor :export_tar_dir, :export_tar_file, :export_tar_prefix
+
+      desc "Import a content view version"
+
+      command_name "import"
+
+      success_message _("Content view imported.")
+      failure_message _("Could not import the content view")
+
+      option(
+        '--export-tar',
+        'EXPORT_TAR',
+        _("Location of export tar on disk.")
+      )
+
+      validate_options do
+        option(:option_export_tar).required
+      end
+
+      build_options
+
+      def execute
+        unless File.exist?(options['option_export_tar'])
+          raise "Export tar #{options['option_export_tar']} does not exist"
+        end
+
+        self.export_tar_file = File.basename(options['option_export_tar'])
+        self.export_tar_prefix = @export_tar_file.gsub('.tar', '')
+        self.export_tar_dir = File.dirname(options['option_export_tar'])
+        untar_export
+
+        export_json = read_json
+
+        cv = content_view(export_json['name'])
+        sync_repositories(export_json['repositories'])
+
+        publish(
+          cv['id'],
+          export_json['major'],
+          export_json['minor'],
+          repos_units(export_json['repositories'])
+        )
+        return HammerCLI::EX_OK
```

Comment:
was this just added? I was about to request it be added but saw it was already here :cat: